packages/mg-revue-api/test/processor.test.js

```diff
@@ -0,0 +1,56 @@
+// Switch these lines once there are useful utils
+const testUtils = require('./utils');
+const processor = require('../lib/processor');
+
+describe('Process', function () {
+    it('Can convert a single post', function () {
+        const fixture = testUtils.fixtures.readSync('api-response.json');
+        const ctx = {
+            result: fixture,
+            options: {
+                addPrimaryTag: 'Newsletter',
+                email: 'person@dummyurl.com',
+                pubName: 'Epic News'
+            }
+        };
+        const response = processor.all(ctx);
+        const posts = response.posts;
+
+        posts.should.be.an.Array().with.lengthOf(2);
+
+        const post = posts[0];
+
+        post.should.be.an.Object().with.properties('url', 'data');
+
+        post.url.should.eql('https://www.getrevue.co/profile/sampleuser/issues/weekly-newsletter-of-sampleuser-issue-1-123456');
+
+        const data = post.data;
+
+        data.slug.should.eql('https://www.getrevue.co/profile/sampleuser/issues/weekly-newsletter-of-sampleuser-issue-1-123456');
```

Comment:
I think `slug` should not include the whole URL, that's why we try to remove it [here](https://github.com/TryGhost/migrate/blob/master/packages/mg-revue-api/lib/processor.js#L180) with regex. Can you check that again?